src/components/WebexAvatar/README.md

```diff
@@ -19,13 +19,15 @@ To see all the different possible states of the Webex Avatar component, you can
 1. Create a component adapter from which the data will be retrieved (See [adapters](../../adapters)). For instance:
 
    ```js
-   const jsonAdapter = new PeopleJSONAdapter(people);
+   const WebexJSONAdapter = {peopleAdapter: new PeopleJSONAdapter(people)};
```

Comment:
I meant to say, this should be a `WebexJSONAdapter`, as in the object; not that you change the name of the variable.
```suggestion
   const jsonAdapter = new WebexJSONAdapter(jsonData);
```